include/oneapi/dpl/pstl/hetero/dpcpp/sycl_traits.h

```diff
@@ -191,22 +250,77 @@ struct __brick_reduce_idx;
 
 } // namespace oneapi::dpl::unseq_backend
 
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::walk_n)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::walk_adjacent_difference)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::transform_reduce)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::reduce_over_group)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::single_match_pred_by_idx)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::multiple_match_pred)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::n_elem_match_pred)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::first_match_pred)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::__create_mask)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::__copy_by_mask)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::__partition_by_mask)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::__global_scan_functor)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::__scan)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::__brick_includes)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::__brick_set_op)
-_ONEDPL_DEVICE_COPYABLE(oneapi::dpl::unseq_backend::__brick_reduce_idx)
+template <typename _ExecutionPolicy, typename _F>
+struct sycl::is_device_copyable<_ONEDPL_SPECIALIZE_FOR(oneapi::dpl::unseq_backend::walk_n, _ExecutionPolicy, _F)>:
+    __are_all_device_copyable<_F> {};
+
+template <typename _ExecutionPolicy, typename _F>
+struct sycl::is_device_copyable<_ONEDPL_SPECIALIZE_FOR(oneapi::dpl::unseq_backend::walk_adjacent_difference, _ExecutionPolicy, _F)>:
+    __are_all_device_copyable<_F> {};
+
+template <typename _ExecutionPolicy, ::std::uint8_t __iters_per_work_item, typename _Operation1, typename _Operation2,
```

Comment:
Do we want to have a little bit of clean up of `::std` in favor of just `std`? there are several places in this PR

Reply:
sure I'll sweep through.

Reply:
done.